src/static/settings.html

```diff
@@ -0,0 +1,70 @@
+<html lang="en">
+
+<head>
+  <meta charset="utf8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <link rel="stylesheet" href="settings.css" type="text/css">
+  <link rel="stylesheet" href="content.css" type="text/css">
+  <link rel="stylesheet" href="hint.css" type="text/css">
+  <link rel="shortcut icon" href="logo/Tridactyl_64px.png" type="img/png">
+  <title>Tridactyl Settings</title>
+</head>
+
+<body class="container">
+  <h1 class="title">Tridactyl Settings</h1>
+  <nav>
+    <ul id="tabbar" class="nav nav-tabs" role="tablist">
+      <li class="nav-item">
+        <a class="nav-link" role="tab" href="#current-settings">Current Settings</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" role="tab" href="#rc-text">.tridactylrc</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" role="tab" href="#about">About</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" role="tab" href="#license">License</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" role="tab" href="#changelog">Changelog</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" role="tab" href="#readme">Readme</a>
+      </li>
+    </ul>
+  </nav>
+  <div id="tab-container" class="tab-content">
+    <div class="tab-panel" role="tabpanel" id="current-settings"></div>
+    <div class="tab-panel" role="tabpanel" id="rc-text">
+      <p>
+        If the button to load from filesystem is not working, check the console
+        to see the error you are getting. Yes, we don't have an error reporting
+        mechanism right now.
+      </p>
+      <button id="btn-save-rc">Save to browser</button>
```

Comment:
On closer inspection - I have no idea what this button does.

I have an RC file with `bind q fillcmdline QQQ` which I can load in using the third button.

If I click the first button, I get "Setting new RC file: bind q..." but it doesn't seem to do anything.

And if the first button is to load the configuration into the browser to make all the binds etc, what on earth does the second button do?